Troupon.Catalog.Api/Startup.cs

```diff
@@ -84,10 +84,6 @@ public void Configure(IApplicationBuilder app, IApiVersionDescriptionProvider ap
       app.UseExceptionHandler("/error");
       app.UseHttpsRedirection();
       app.UseSerilogRequestLogging();
-
```

Comment:
We probably want to keep that.

I've reated a MsSql docker Setup in git at:
./troupon.catalog/infratructure/docker

Then in Visual Studio you can override your local connexion string so you project start by creating/migrating the database for you with EntityFramework

Reply:
- [ ] bring migration this back ?

Reply:
Are we using that right now? Because if not, I don't see a reason to add it there. I feel like we could evolve the API step by step. If we are not using database operations at the current moment, we are bringing unnecessary extra complexity to setup and run the environment

Reply:
Great point ! We won't implement the persistence in near future.

Maybe we could do as I did with the Authentication and "Park it". We could create a FeatureFlag like
```
{
    "FeatureManagement": {
       "Infra.Features.Persistance.MigrationOnStartup.Disable" : true
    }
}
```

So that the Statup will be fast while development (and no need to stat Docker and all this stuff)

Reply:
Sounds great! Since we will work with repository patterns in the next sprint, we could address in the planning what approach we could take

Reply:
All good with that !